commands/sandbox.go

```diff
@@ -39,7 +39,7 @@ const (
 	// Minimum required version of the sandbox plugin code.  The first part is
 	// the version of the incorporated Nimbella CLI and the second part is the
 	// version of the bridge code in the sandbox plugin repository.
-	minSandboxVersion = "3.0.4-1.2.0"
+	minSandboxVersion = "3.0.5-1.2.0"
```

Comment:
Orthogonal to the purpose of the PR, but, meanwhile, there is a new `nim` that fixes some vulnerabilities.